docs/BasicCommunication/OnSystemCapabilityUpdated/assets/OnSystemCapabilityUpdated_REMOVED.puml

```diff
@@ -2,18 +2,15 @@
 
 participant Mobile
 participant Core
-participant AppServiceManager
 participant HMI
 
 note left of Mobile: Mobile app has \nmultiple published \napp services
 
 Mobile -> Core : Disconnect
 Core -> Core : UnregisterApplication
-Core -> AppServiceManager : UnpublishServices(app_id)
 
 loop For Every Published Service On Device
```

Comment:
perhaps `For every service published by Application` would be more accurate, since you could unregister an application without disconnecting.